system-int-test/test/cmr/system_int_test/ingest/granule_bulk_update/granule_bulk_update_volume_test.clj

```diff
@@ -0,0 +1,72 @@
+(ns cmr.system-int-test.ingest.granule-bulk-update.granule-bulk-update-volume-test
+  "CMR granule bulk update volume integration tests"
+  (:require
+   [cheshire.core :as json]
+   [clojure.test :refer :all]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.message-queue.test.queue-broker-side-api :as qb-side-api]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.core :as data-core]
+   [cmr.system-int-test.data2.granule :as granule]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(def base-request {:name "large update request"
+                   :operation "UPDATE_FIELD"
+                   :update-field "OPeNDAPLink"
+                   :updates []})
+
+(defn update-instruction
+  "Generate an update instruction."
+  [ur]
+  [ur (str "https://file.example.nasa.gov/" ur)])
+
+;; This test is potentially unstable on systems with smaller amounts of
+;; memory and can take multiple minutes to complete. Test with caution
```

Comment:
There is no need to test with `501` granules. Testing with `granule-bulk-update-chunk-size` of 2 and 3 granules is sufficient.